engines/engine.cpp

```diff
@@ -310,37 +310,49 @@ void initGraphicsModes(const Graphics::ModeList &modes) {
 	g_system->initSizeHint(modes);
 }
 
-void initGraphics(int width, int height, const Graphics::PixelFormat *format) {
-
-	g_system->beginGFXTransaction();
+int initGraphicsAny(const Graphics::ModeWithFormatList &modes) {
```

Comment:
Please add some documentation for the method.

Reply:
Done